.github/workflows/build.yml

```diff
@@ -79,60 +91,79 @@ jobs:
       logging_level: ${{ inputs.logging_level }}
       build_note: ${{ inputs.build_note }}
       run_export: ${{ inputs.run_export }}
+      image_tag: ${{ needs.health_check.outputs.tag }}
```

Comment:
this seemed better than just defining it over and over again, and this is the simplest way to define a re-usable variable within the action as far as I can tell (`env` vars are not accessible when calling other workflows. So couldn't do ${{ env.tag }})

Reply:
and in the longer-term, the repeated `image_tag: ${{ needs.health_check.outputs.tag }}` lines would be avoided by not having product-specific workflows at all (e.g. `build.yml` calls `dcpy.builds` scripts with some inputs like product name)